web/pgadmin/browser/__init__.py

```diff
@@ -784,6 +790,14 @@ def reset_master_password():
     return make_json_response(data=get_crypt_key()[0])
 
 
+def check_keyring_present():
+    import keyring
```

Comment:
keyring is already imported.

Reply:
Removed this function as not using it anymore